NestJS example build errors

When I try and start the NestJS example, it fails with various TypeScript errors:

**Node:** `v20.17.0`

```
10:50:36  ~/Code/arcjet-js/examples/nodejs-nestjs git:(main) npm run start

> nest-js@0.0.1 start
> nest start

../../protocol/convert.ts:12:8 - error TS2835: Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './index.js'?

12 } from "./index";
          ~~~~~~~~~
../../protocol/convert.ts:34:8 - error TS2835: Relative import paths need explicit file extensions in ECMAScript imports when '--moduleResolution' is 'node16' or 'nodenext'. Did you mean './index.js'?

34 } from "./index";
          ~~~~~~~~~
../../protocol/convert.ts:63:13 - error TS2322: Type 'ArcjetMode' is not assignable to type 'never'.

63       const _exhaustive: never = mode;
               ~~~~~~~~~~~
../../protocol/convert.ts:84:13 - error TS2322: Type 'ArcjetEmailType' is not assignable to type 'never'.

84       const _exhaustive: never = emailType;
               ~~~~~~~~~~~
../../protocol/convert.ts:124:13 - error TS2322: Type 'ArcjetStack' is not assignable to type 'never'.

124       const _exhaustive: never = stack;
                ~~~~~~~~~~~
../../protocol/convert.ts:141:13 - error TS2322: Type 'ArcjetRuleState' is not assignable to type 'never'.

141       const _exhaustive: never = stack;
                ~~~~~~~~~~~
../../protocol/convert.ts:181:13 - error TS2322: Type 'ArcjetConclusion' is not assignable to type 'never'.

181       const _exhaustive: never = conclusion;
                ~~~~~~~~~~~

Found 7 error(s).
```

--- a/protocol/convert.ts
+++ b/protocol/convert.ts
@@ -9,7 +9,7 @@ import type {
   ArcjetSlidingWindowRateLimitRule,
   ArcjetShieldRule,
   ArcjetSensitiveInfoRule,
-} from "./index";
+} from "./index.js";
 import {
   ArcjetAllowDecision,
   ArcjetBotReason,
@@ -31,7 +31,7 @@ import {
   ArcjetStack,
   ArcjetIpDetails,
   ArcjetSensitiveInfoReason,
-} from "./index";
+} from "./index.js";
 import type { IpDetails } from "./proto/decide/v1alpha1/decide_pb.js";
 import {
   BotV2Reason,

